src/pages/wallet-import-keystore/wallet-import-keystore.ts

```diff
@@ -32,6 +34,12 @@ export class WalletImportKeystorePage {
    * Set to true if the keystore could not be decrypted with the password.
    */
   passwordIsInvalid: boolean = false;
+  filePasswordIsInvalid: boolean = false;
+  keystoreFileIsInvalid: boolean = false;
+  keystoreFileInvalidData: boolean = false;
+  importedFilename: string = "";
+  passwordFile: string = "";
```

Comment:
Is this the password to use for the imported file? Or the password used for both the string keystore and the file keystore? Either way I think this variable name is somewhat confusing.

Reply:
True, will be fixed next request in clear keystore file & clipboard variables